src/i18n/Localizable.xcstrings

```diff
@@ -8486,6 +8541,7 @@
       }
     },
     "Fast" : {
+      "comment" : "a bit ambiguous word. example needed :(",
```

Comment:
"Fast" as speedy. Used to ask user if the cursor need to be moved slowly or fastly